app.js

```diff
@@ -1,10 +1,145 @@
 /* Imports */
-// import { getRandomItem } from './utils.js';
+import { getRandomItem } from './utils.js';
 
 /* State */
+// let shell = ['shell-1', 'shell-2', 'shell-3'];
+let pearl = ['pearl-1', 'pearl-2', 'pearl-3'];
+let guess = ['guess1', 'guess2', 'guess3'];
+const display1 = document.getElementById('display-1');
+const display2 = document.getElementById('display-2');
+const display3 = document.getElementById('display-3');
 
-/* Actions */
-function loadPage() {}
+const guess1 = document.getElementById('guess-1');
+const guess2 = document.getElementById('guess-2');
+const guess3 = document.getElementById('guess-3');
+const shell1 = document.getElementById('shell-1');
+const shell2 = document.getElementById('shell-2');
+const shell3 = document.getElementById('shell-3');
+const playAgainButton = document.getElementById('play-again-button');
+// const totalDisplay = document.getElementById('total-display');
+// const winsDisplay = document.getElementById('wins-display');
+// const lossesDisplay = document.getElementById('losses-display');
+// let gameState = '';
+let computer = '';
+
+// let guess = [guess1, guess2, guess3];
+
+function loadPage() {
+    displayShells();
+    pearl.classList.remove('hidden');
+    // gameState = 'guess';
+}
+
+guess1.addEventListener('click', () => {
+    guess = guess1;
```

Comment:
the variable ```guess``` is an array of guesses. you'd either wanna leave it as a blank piece of state rather than the array ```let guess = ''``` so you can store that value and do things with it, or inside this event listener, use ```guess[0]``` the access the first thing in the array.